doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_lifecycle.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _ug_nrf54h20_architecture_lifecycle:
 
 nRF54H20 lifecycle states
```

Comment:
Line 53 of this file:

"Black arrows illustrate forward unauthenticated transitions and red arrows illustrate backward authenticated transitions".

The issue - different coloring schema is applied to diagram above

Reply:
Edited to work around the issue